### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by avtikhon**
Added initial information in README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
 # tarantool-qa
+
+Repository for QA activities and issues of the [tarantool/tarantool](https://github.com/tarantool/tarantool) project.
+
+Main target of this repository to detach internal QA activities from [Tarantool issues](https://github.com/tarantool/tarantool/issues) interesting to Tarantool users.
+This repository uses for issues on:
```
Comment (Totktonada) [inline README.md:6]: Please, split paragraphs with an empty line: this way it looks better as in text as well as rendered.

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+ * Tests fixes for issues in tests.
+ * Tests issues reproducers.
+ * Setup test infrastructure, like [Github Actions](https://github.com/tarantool/tarantool/actions).
```
Comment (Totktonada) [inline README.md:9]: I would adjust the wording a bit:

The repository is used for tasks of the following kinds:

* Testing problems (say, a badly written test). Logs, reproducers and so on are here.
* Testing infrastructure problems (say, network problem with some external resource).
* Testing enhancements and plans (for example, transition to [Github Actions](https://github.com/tarantool/tarantool/actions)).

**Review by Totktonada — COMMENTED** [review]
Some wording tweaks.

File: README.md (hunk 1 above)
Comment (avtikhon) [reply README.md:7]: Sure, done.

File: README.md (hunk 2 above)
Comment (avtikhon) [reply README.md]: Right, I'll use your variant.

Comment (Totktonada) [reply README.md]: Still 'This repository uses for issues on'. It looks grammatically incorrect: the repository *is used for*, not *uses for*.

Comment (avtikhon) [reply README.md]: Corrected.

**Review by Totktonada — APPROVED** [review]

**Pull request merged**